jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1066,6 +1072,14 @@ public void parse(final String rawarg, String source)
             return;
         }
 
+        // to override default http://central.maven.org/maven2/
+        if (arg.startsWith("--maven-base-url="))
+        {
+            this.mavenBaseUrl = Props.getValue(arg);
+            run = false;
```

Comment:
Note sure that setting this option alone is worth a `run = false` - but then I can't think of why it would be used in a normal run?

Reply:
copy/paste issue...